src/styles/main.css

```diff
@@ -20,6 +20,16 @@
     url("./../fonts/roboto-mono/roboto-mono-v13-latin-300.woff") format("woff");
 }
 
+/** Use Arial for "." and "," instead of Roboto Mono. */
+@font-face {
+  font-family: 'Roboto Mono';
+  font-style: normal;
+  font-weight: 300;
+  font-display: swap;
+  src: local("Arial");
+  unicode-range: U+002E, U+002C;
+}
```

Comment:
> @buberdds is the "after" screenshot up to date? in a preview URL spacing looks weird

It is up to date. I tried it in Firefox now, and it looks like your screenshot. So this code isn't working consistently :shrug:

Reply:
Updated to use Rubik and it fixes my Firefox
![localhost_3000_account_oasis1qrtyn2q78jv6plrmexrsrv4dh89wv4n49udtg2km](https://github.com/user-attachments/assets/9e24557e-44b3-476f-87b1-5eec4099af96)

Try again?

Reply:
looks much better